### DISCUSSION TITLE
Outputting PETSC -log_view data

### DISCUSSION THREAD

**Discussion by TheBEllis** [mfem/mfem#3950] [Q&A] [answered]
Hi MFEM team,
I'm testing out some PETSC solvers currently, and am looking to use the PETSC -log_view flag to get some more detailed info. I am initialising petsc with an options file that specifies all my flags. However log_view does not seem to be able to output to a file or stdout. Any suggestions are appreciated:)

**Comment by tzanio** [comment]
ping: @stefanozampini

**Comment by jandrej** [comment, marked as answer]
```
$ cat rc_ex1p
....
-pc_gamg_use_sa_esteig 0
-mg_levels_pc_type sor

-log_view
```

```
% ./ex1p -petscopts rc_ex1p
...
------------------------------------------------------------------ PETSc Performance Summary: ------------------------------------------------------------------
```

Works as expected.

**Reply by TheBEllis** [reply]
Thank you for the pointer, I went back and compared what I had to ex1p. It was a silly mistake on my part, I wasn't calling the finalisePETSC method, hence no output!
